precisely/comparison_matchers.py

```diff
@@ -45,3 +45,22 @@ def match(self, actual):
 
     def describe(self):
         return "{0} {1!r}".format(self._operator_description, self._value)
+
+
+def close_to(value, delta):
+    return IsCloseToMatcher(value, delta)
+
+
+class IsCloseToMatcher(object):
+    def __init__(self, value, delta):
+        self._value = value
+        self._delta = delta
+
+    def match(self, actual):
+        if self._value - self._delta <= actual <= self._value + self._delta:
+            return matched()
+        else:
+            return unmatched("was {0!r} ({1!r} away from {2!r})".format(actual, abs(self._value - actual), self._value))
```

Comment:
Thinking about this a bit more, we should be consistent in how we determine the gap. If we're requiring support for `abs()` (which I don't think we were in previous commits), then we should use the same value throughout i.e.

```python
difference = abs(self._value - actual)
if difference > self._delta:
    return unmatched("was {0!r} ({1!r} away from {2!r})".format(actual, difference, self._value))
else:
    return matched()
```

Apologies for the back and forth! Other than this, I think this is good to go.

Reply:
The `abs` only went in to avoid negative distances - will update the rest of the code.